components/HTTP/src/Websocket.cpp

```diff
@@ -47,423 +46,560 @@ static esp_err_t _json_parse_hex(cJSON* root, std::uint8_t *buf,
     return ESP_OK;
 }
 
-void websocketRadioRx(const std::uint8_t* buffer, std::size_t length)
+esp_err_t bytesToHex(std::uint8_t* buf, char* dst, std::size_t len)
 {
-    /*
-    char payload[500];
-    esp_err_t err;
-    err = util::decode((std::uint8_t*)buffer, payload);
-    if (err) {
-        ESP_LOGE("TEST", "error decoding the value");
+    for (size_t i = 0; i < len; i++) {
+        const uint8_t upper = (buf[i] & 0xf0) >> 4;
+        const uint8_t lower = (buf[i] & 0x0f);
+        dst[i] = buf[upper];
+        dst[i + 1] = buf[lower];
```

Comment:
```suggestion
   static const char hexmap[] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};

    for (size_t i = 0; i < len; i++) {
        const uint8_t upper = (buf[i] & 0xf0) >> 4;
        const uint8_t lower = (buf[i] & 0x0f);
        dst[i] = hexmap[upper];
        dst[i + 1] = hexmap[lower];
```

hexmap maps `lower` and `upper` to characters

Reply:
```suggestion
    for (size_t i = 0; i < len; i++) {
        const uint8_t upper = (buf[i] & 0xf0) >> 4;
        const uint8_t lower = (buf[i] & 0x0f);
        dst[i * 2] = buf[upper];
        dst[i * 2+ 1] = buf[lower];
```